cli/scripts/0_swarm_secrets.js

```diff
@@ -14,38 +14,37 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-const lightning = require('lnrpc-node-client')
 const exec = require('executive')
 const chalk = require('chalk')
 const generator = require('generate-password')
-const homedir = require('os').homedir()
```

Comment:
@jasonbukowski Was this the homedir call that worked for you?

Reply:
Yes

Reply:
I hadn't updated that, or anything else of that nature, in the cli directory. The main goal here was to just migrate over to the lnrpc-node-client package. I will address specific things like this as I work through the cli script init process.

Reply:
Its removal in this instance was just to satisfy an 'unused package' eslint rule.